scene/2d/tile_map.h

```diff
@@ -143,6 +143,10 @@ class TileMap : public Node2D {
 	int get_layer_y_sort_origin(int p_layer) const;
 	void set_layer_z_index(int p_layer, int p_z_index);
 	int get_layer_z_index(int p_layer) const;
+	void set_layer_collision_enabled(int p_layer, bool p_enabled);
+	bool is_layer_collision_enabled(int p_layer) const;
```

Comment:
Regardless if we add the occlusion feature to `TileMap` this code is unrelated and shouldn't be in this PR IMO